core/modules/server/routes/get-file.js

```diff
@@ -38,7 +38,7 @@ exports.handler = function(request,response,state) {
 		} else {
 			status = 200;
 			content = content;
-			type = $tw.config.fileExtensionInfo[extension] || "application/octet-stream";
+			type = $tw.config.fileExtensionInfo[extension].type || "application/octet-stream";
```

Comment:
Might we need to guard against `$tw.config.fileExtensionInfo[extension]` being undefined?

Reply:
Indeed we do